From: Muhammad Usama Anjum <usama.anjum@collabora.com>
Subject: selftests/mm: config: add missing configs
Date: Fri, 12 Jan 2024 12:21:40 +0500

Add configurations which are needed for
- hugetlb-read-hwpoison
- ksm_functional_test and ksm_test

Link: https://lkml.kernel.org/r/20240112072144.620098-7-usama.anjum@collabora.com
Signed-off-by: Muhammad Usama Anjum <usama.anjum@collabora.com>
Cc: Shuah Khan <shuah@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/mm/config |    3 +++
 1 file changed, 3 insertions(+)

--- a/tools/testing/selftests/mm/config~selftests-mm-config-add-missing-configs
+++ a/tools/testing/selftests/mm/config
@@ -7,3 +7,6 @@ CONFIG_TEST_HMM=m
 CONFIG_GUP_TEST=y
 CONFIG_TRANSPARENT_HUGEPAGE=y
 CONFIG_MEM_SOFT_DIRTY=y
+CONFIG_MEMORY_FAILURE=y
+CONFIG_HWPOISON_INJECT=y
+CONFIG_KSM=y
_
